01week/rockPaperScissors.js

```diff
@@ -9,11 +9,59 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+	const rockPaperScissorsArray = ['rock', 'paper', 'scissors'];
 
-  // Write code here
+	// get rid of whitespace and convert to lower case before beginning to check
+	hand1 = hand1.trim().toLowerCase();
+	hand2 = hand2.trim().toLowerCase();
 
-}
+	// only accept rock, paper, or scissors
+	if (rockPaperScissorsArray.indexOf(hand1) < 0 || rockPaperScissorsArray.indexOf(hand2) < 0) {
+		return "Please choose either rock, paper, or scissors.";
+		getPrompt();
```

Comment:
You don't have to call getPrompt() within your function, it automatically runs beneath.